src/lib/components/Form/Field/19_ContactsField.svelte

```diff
@@ -52,6 +53,24 @@
     validationResult?.find(({ index }) => index === undefined)
   );
 
+  const autoFillUserDetails = async () => {
+    addItem(new MouseEvent('click'));
```

Comment:
The event can be omited i guess. I'm not even sure if the preventDefault in the addItem is still needed. So the `evt` argument could be removed completly.

Reply:
Removed it